components/table/style/size.less

```diff
@@ -124,7 +124,6 @@
     .@{table-prefix-cls}-footer {
       border: 0;
       border-top: @border-width-base @border-style-base @border-color-split;
-      border-right: @border-width-base @border-style-base @border-color-split;
```

Comment:
plz do not modify less file under style folder except patch.less, they are all synced from antd react.